### ISSUE TITLE
Where is the implementation of orthogonal regularization loss(Eq. 12) in your paper?

### ISSUE THREAD

**Issue by nihaomiao** [carrierlxk/COSNet#16]
Hi, Xiankai,

Thanks very much for sharing your code!

In your paper, you mentioned that you add orthogonal regularization into your loss function (Equation 12). However, in your training code, I only find the L1 loss between prediction and ground truth (details can be found in [here](https://github.com/carrierlxk/COSNet/blob/dbae2e8d39edd286e0fd90f3fbb29b73e9ec205d/train_iteration_conf.py#L178) and [here](https://github.com/carrierlxk/COSNet/blob/dbae2e8d39edd286e0fd90f3fbb29b73e9ec205d/train_iteration_conf.py#L420)).  So could you please explain why this L1 loss can achieve the effect of orthogonal regularization?

Thanks very much for your time!!!
